integrations/deploy.yaml

```diff
@@ -542,3 +542,48 @@
     group: ''
     distro: ''
   quick_start: 2
+- id: deploy-freebsd
+  meta:
+    name: FreeBSD
+    link: 'https://www.freebsd.org/'
+    categories:
+      - deploy.operating-systems
+    icon_filename: 'freebsd.jpg'
+  most_popular: true
+  keywords:
+    - freebsd
+  install_description: |
+    ## Install dependencies
+    Please install the following packages using the command below:
+
+    ```pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof liblz4 libuv json-c cmake gmake```
+    This step needs root privileges. Please respond in the affirmative for any relevant prompts during the installation process.
+    
+    Run the following command on your node to install and claim Netdata:
+  methods:
+    - method: wget
+      commands:
+        - channel: nightly
+          command: >
+            wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --nightly-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+        - channel: stable
+          command: >
+            wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --stable-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+    - method: curl
+      commands:
+        - channel: nightly
+          command: >
+            curl https://my-netdata.io/kickstart.sh > /tmp/netdata-kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --nightly-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+        - channel: stable
+          command: >
+            curl https://my-netdata.io/kickstart.sh > /tmp/netdata-kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --stable-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+  additional_info: *ref_containers
```

Comment:
Let's probably add this as additional info
> Netdata can also be installed via [FreeBSD ports](https://www.freshports.org/net-mgmt/netdata).

Reply:
Done